### ISSUE TITLE
bug with histogram

### ISSUE THREAD

**Issue by Jorge-hercas** [JohnCoene/echarts4r#346]
Hello, goodnight. I recently made an app in shiny that generates histograms and I noticed that there is a small bug when generating a histogram, as it looks like this:

<img width="635" alt="Captura de Pantalla 2021-07-17 a la(s) 22 29 58" src="https://user-images.githubusercontent.com/70007745/126054718-6c7cf50b-520a-4272-8360-e86a9c13be49.png">

Thanks for reading me

**Comment by JohnCoene** [comment]
Could you provide me with a reproducible example so I can look into it?

**Comment by Jorge-hercas** [comment]
Yup, the complete code, data and app are here: https://github.com/Jorge-hercas/leaflet_decilesapp

**Comment by helgasoft** [comment]
it's a [bug in ECharts](https://github.com/apache/echarts/issues/15189).
Workarounds - set ```scale=FALSE```, or ```breaks='Sturges'```.
